crates/bevy_ecs/src/system/system.rs

```diff
@@ -1,3 +1,14 @@
+/// The main building block in the Bevy ECS app model.
+///
+/// Systems are usually written as normal rust functions. Then the `.system()` trait
+/// extension method is called to make them be Bevy systems.
+///
+/// System functions can have parameters, through which one can query and
+/// mutate Bevy ECS states.
+/// See [The Bevy Book](https://bevyengine.org/learn/book/introduction/) for more information.
+///
+/// Systems are usually run in parallel, and the execution order is not deterministic.
+/// If you want more fine-grained control for order, see [`AppBuilder::add_system_to_stage`].
```

Comment:
Perhaps also mention `.laberl()` + `.after()`/`.befor()` for same stage ordering

Reply:
This is just some docs I moved from `add_system`. It may be a good idea though to add an overview of the functionality of systems on the module level, since there are no easy to find methods.